src/lib/Zikula/Core/AbstractBundle.php

```diff
@@ -206,6 +207,22 @@ public function getContainer()
     }
 
     /**
+     * Add the bundle's stylesheet to the page assets
+     * @param string $name
+     */
+    public function addStylesheet($name = 'style.css')
+    {
+        try {
+            $styleSheet = $this->getContainer()->get('zikula_core.common.theme.asset_helper')->resolve('@' . $this->getName() . ":css/$name");
+        } catch (\InvalidArgumentException $e) {
+            $styleSheet = '';
+        }
+        if (!empty($styleSheet)) {
+            $this->container->get('zikula_core.common.theme.assets_css')->add([$styleSheet => AssetBag::WEIGHT_DEFAULT]);
```

Comment:
My point was theme stylesheets should be loaded after other ones so that it becomes easier to override and/or customise module layouts.